src/services/blocks/BlocksService.ts

```diff
@@ -532,7 +532,9 @@ export class BlocksService extends AbstractService {
 		if (withdrawEvent.length > 0 && withdrawEvent[0].data) {
 			const dataArr = withdrawEvent[0].data.toJSON();
 			if (Array.isArray(dataArr)) {
-				const fee = (dataArr as Array<number>)[dataArr.length - 1];
+				const fee = this.sanitizeFee(
+					(dataArr as Array<string>)[dataArr.length - 1]
```

Comment:
Did the shape change here or do we want to have it seen as `string` for the `sanitizeFee`/`isHex` call?

Reply:
So the shape was inaccurately type casted for fees (technically number or string work here for BN.js but I changed it to its proper type to be more accurate). The type returned from `.toJSON()` should be a string, because any u128/u64 value interpreted by polkadot-js will be returned as a string.